sw/airborne/modules/relative_localization_filter/relative_localization_filter.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) Mario Coppola
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/relativelocalizationfilter/relativelocalizationfilter.h"
+ * @author Mario Coppola
+ * Relative Localization Filter for collision avoidance between drones
+ */
+
+#ifndef RELATIVE_LOCALIZATION_FILTER_H
+#define RELATIVE_LOCALIZATION_FILTER_H
+
+#ifndef RL_NUAVS
+#define RL_NUAVS 5 // Maximum expected number of UAVs
+#endif
+
+#include "discrete_ekf.h"
+
+extern discrete_ekf ekf_rl[RL_NUAVS - 1];
```

Comment:
why is this defined here? Is this parameter used outside of the relative_localization_filter source file? Only include things needed by other files in the header.

Reply:
The idea is that, if desired, this can be directly read out from other modules so that they can do something with the info. I could also change to an ABI message though, if that's preferred in paparazzi.